docs/body.html

```diff
@@ -167,79 +156,71 @@ <h5 class="date">September 9, 2021</h5>
             <a href="https://discourse.matplotlib.org/t/matplotlib-awarded-czi-eoss-grant-cycle-4/22300" class="link--offsite">Matplotlib awarded a CZI EOSS Cycle 4 grant</a>
           </div>
 
-        <!-- END NEWS ITEMS -->
-        <!-- link to discourse -->
-        <div class="news__discourse-link">
-          <div class="rule"></div>
-          <a
-            href="https://discourse.matplotlib.org/c/announce/14"
-            class="link--offsite"
-            >Older Announcements</a
-          >
+          <!-- END NEWS ITEMS -->
+          <!-- link to discourse -->
+          <div class="news__discourse-link">
+            <div class="rule"></div>
+            <a
+              href="https://discourse.matplotlib.org/c/announce/14"
+              class="link--offsite">Older Announcements</a>
+          </div>
```

Comment:
This block was dedented one level, and so the last `</div>` was probably accidentally missed.